storage/libradb/src/ledger_store/ledger_info_test.rs

```diff
@@ -48,18 +48,21 @@ proptest! {
             .filter(|info| info.ledger_info().next_validator_set().is_some())
             .cloned()
             .collect();
-        let start_epoch = ledger_infos_with_sigs.first().unwrap().ledger_info().epoch();
-        let ledger_version = ledger_infos_with_sigs.last().unwrap().ledger_info().version();
-        prop_assert_eq!(
-            &store.get_epoch_change_ledger_infos(start_epoch, ledger_version).unwrap(),
-            &epoch_ledgers
-        );
-
-        // verify get_epoch()
         let epoch_change_versions: Vec<_> = epoch_ledgers.iter().map(|li| {
             let ledger_info = li.ledger_info();
             (ledger_info.epoch(), ledger_info.version())
         }).collect();
+        let start_epoch = ledger_infos_with_sigs.first().unwrap().ledger_info().epoch();
+        let latest_ledger_info = ledger_infos_with_sigs.last().unwrap().ledger_info();
+        let ledger_version = latest_ledger_info.version();
+        let proof = store.get_epoch_change_ledger_infos(start_epoch, ledger_version).unwrap();
+        let expected_proof = epoch_ledgers
+                .into_iter()
+                .filter(|li| li.ledger_info().version() < ledger_version)
+                .collect::<Vec<_>>();
```

Comment:
Is it okay to simply remove the last one instead of doing this filtering. Same below.

Reply:
it doesn't work. If latest_ledger_info is in the middle of one epoch, we don't pop the last one.

Reply:
it is unclear to me what this is proving aren't these two LIs equal? Maybe it should be version < version but epoch > epoch